demo/src/pages/index.js

```diff
@@ -38,10 +41,28 @@ const liveCodeSample = `
 <Button variant={Button.VARIANT.PRIMARY} onClick={() => alert('Hello!')}>Hello!</Button>
 `;
 
+const gdprConsentCookieName = 'yummyCookie'
+
+const options = { expires: 1 /* day */ };
+
 const IndexPage = ({ data }) => {
   const { layout, siteMetadata } = data.site;
   const [searchTerm, setSearchTerm] = useState('');
   const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [cookieConsent, setCookieConsent] = useState(Cookies.get(gdprConsentCookieName) === 'true'); 
+
+  const previousCookieConsent = usePrevious(cookieConsent);
+
+  const handleCookieConsent = (answer) => {
+    Cookies.set(gdprConsentCookieName, String(!!answer), options);
+    answer && setCookieConsent(true);
+  } 
+
+  useEffect(()=> {
+      if (cookieConsent && previousCookieConsent === false) {
```

Comment:
Can you run prettier on this file?

I wonder if your editor has turned prettier off or something. I've noticed a couple files that seem to be mis-styled. Could you double check to make sure prettier is formatting your files properly?

Reply:
Yeah for some reason vscode hasn't been formatting my files on save and I've tried a bunch of things to fix it. I've just been running prettier from the command line.